scripts/src/chartprreview/chartprreview.py

```diff
@@ -105,7 +138,15 @@ def check_owners_file_against_directory_structure(
         sys.exit(1)
 
 
-def verify_signature(directory, category, organization, chart, version):
+def verify_signature(category, organization, chart, version):
```

Comment:
Removing directory here seems fine, but I'm wondering: should we have instead leveraged the directory to return an error?

I'm thinking that typically this directory is where PR "artifacts" live... that is, things that typically make it back to the user in the PR as a comment indicating what went wrong.

We covered that directory here is likely unused, but should we have used that directory to communicate some error in this block back to the user instead of removing it?

Reply:
I understand what you mean.

Currently the way errors are handled (at least in the rest of this file) is by writing them to the logs, then exiting with a return code of 1:
```
            write_error_log(directory, msg)
            sys.exit(1)
```

I'm not a fan of this approach, because it means having `sys.exit` deep inside multiple function calls. At some point we should probably review it. For instance, when we harmonize the way github comments are written across the different certification projects.

Also, at this moment I don't see where the artifact directory is used. Besides writing error logs, it seems to be adding some other file metadata (in `prartifact.py`). But is it made eventually available to the user ? As a directory to download ? I can't find this.

So, back to your initial comment, regarding this function. Currently it never fails AFAICT. I can add some error logs and `sys.exit` if the check fails.

Reply:
> I'm not a fan of this approach, because it means having sys.exit deep inside multiple function calls. At some point we should probably review it.

100%. Couldn't agree more.

With this in mind, we can re-evaluate whether or not we need to re-add the directory here in the future when we normalize error messages. I'll mark this topic as resolved.